plugins/modules/systemd_facts.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2025, Marco Noce <nce.marco@gmail.com>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: systemd_facts
+short_description: Gather C(systemd) unit facts.
```

Comment:
All descriptions should have proper sentences in them, *except* `short_description`.
```suggestion
short_description: Gather C(systemd) unit facts
```